### PR TITLE
add extensions to izanagiwki

### PR DESCRIPTION
**Pull request by Reception123**
As no one seems to be handling the request i'll do it. Could someone please mark it as done on the page (i am currently blocked)?

### CODE REVIEW

**Comment by Reception123** [comment]
Conflicts. Now #696

**Comment by Reception123** [comment]
Woops.

File: roles/mediawiki/files/localsettings/LocalExtensions.php [hunk 1]
```diff
@@ -210,6 +210,13 @@
 		break;
 	case 'hmwiki':
 		break;
+	case 'izanagiwiki':
+		require_once( "$IP/extensions/EmbedVideo/EmbedVideo.php" );
+		require_once( "$IP/extensions/MultimediaViewer/MultimediaViewer.php" );
+		require_once( "$IP/extensions/MultiUpload/MultiUpload.php" );
+		break;
+
```
Comment (addshore) [inline roles/mediawiki/files/localsettings/LocalExtensions.php]: please do not add double new lines! or any new lines here at all!

**Comment by Reception123** [comment]
Conflicts?

**Comment by Reception123** [comment]
Now #704

**Pull request closed without merging**